packages/universal-wallet-fastify-plugin/src/__tests__/content.test.ts

```diff
@@ -0,0 +1,100 @@
+import supertest, { SuperTest, Test } from 'supertest';
+import Ajv from 'ajv';
+
+import { getFastifyWithWalletOptions, walletOptions } from './utils';
+import { case2 } from '../__fixtures__/verifiableCredentials';
+import { UniversalWalletFastifyInstance } from '../types';
+import vcSchema from '../schemas/verifiable-credential.json';
+import presentationSubmission from '../__fixtures__/presentationSubmissions/case-0.json';
+
+let api: SuperTest<Test>;
+let fastify: UniversalWalletFastifyInstance;
+
+beforeAll(async () => {
+  fastify = getFastifyWithWalletOptions(walletOptions);
+  await fastify.ready();
+  api = supertest(fastify.server);
+});
+
+afterAll(() => {
+  fastify.close();
+});
+
+describe('content endpoint', () => {
+  describe('vc content', () => {
+    it('creates the credential', async () => {
+      const response = await api.post('/accounts/123/wallet/contents').send({
+        content: case2,
+      });
+      expect(response.status).toBe(200);
+      expect(response.body).toEqual({});
+    });
+
+    it('should filter out not allowed properties', async () => {
```

Comment:
just noting here that as we iterate on this, we should consider returning a 400 error instead of dropping properties that are not in the whitelist.

No changes required